jss.spec

```diff
@@ -126,7 +128,12 @@ modutil -dbdir /etc/pki/nssdb -chkfips true | grep -q enabled && export FIPS_ENA
 ./build.sh \
     %{?_verbose:-v} \
     --work-dir=%{_vpath_builddir} \
+    --prefix-dir=%{_prefix} \
+    --include-dir=%{_includedir} \
     --lib-dir=%{_libdir} \
+    --sysconfig-dir=%{_sysconfigdir} \
+    --share-dir=%{_datadir} \
+    --cmake=%__cmake \
```

Comment:
Is there a reason that there are no curly braces here? I just tried a test on the command line to see what happens when you omit them and it turns out it gives the same value:
```
rpm --eval "some text printed on %{__cmake}"
some text printed on /usr/bin/cmake

rpm --eval "some text printed on %__cmake"
some text printed on /usr/bin/cmake
``` 

Could we standardise one way or the other though please, as it is certainly curious to the inexperienced observer!

Reply:
That's actually how the `%cmake` macro was defined (i.e. with inconsistent usage of curly braces), but I've changed it since it doesn't seem to cause any problem.